JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -78,42 +78,45 @@ module Microsoft.ApplicationInsights {
          * Add a telemetry item to the send buffer
          */
         public send(envelope: Telemetry.Common.Envelope) {
-            
-            // if master off switch is set, don't send any data
-            if (this._config.disableTelemetry()) {
-                // Do not send/save data
-                return;
-            }
+            try {
+                // if master off switch is set, don't send any data
+                if (this._config.disableTelemetry()) {
+                    // Do not send/save data
+                    return;
+                }
         
-            // validate input
-            if (!envelope) {
-                _InternalLogging.throwInternalUserActionable(LoggingSeverity.WARNING, "Cannot send empty telemetry");
-                return;
-            }
+                // validate input
+                if (!envelope) {
+                    _InternalLogging.throwInternalUserActionable(LoggingSeverity.WARNING, "Cannot send empty telemetry");
+                    return;
+                }
 
-            // ensure a sender was constructed
-            if (!this._sender) {
-                _InternalLogging.warn("No sender could be constructed for this environment, payload will be added to buffer." + Serializer.serialize(envelope));
-                return;
-            }
+                // ensure a sender was constructed
+                if (!this._sender) {
+                    _InternalLogging.warn("No sender could be constructed for this environment, payload will be added to buffer." + Serializer.serialize(envelope));
```

Comment:
This warning looks outdated, the payload doesn't get added to the buffer in this case.